doc/run/repl.md

```diff
@@ -1,16 +1,18 @@
-# Run SQLFlow REPL
+# Run SQLFlow command-line tool
 
-In addition to building SQLFlow into a gRPC server, accessed via Jupyter Notebook, we could also build it into a command line program, whose `main` function reads SQL statements from the console, evaluates them by calling SQLFlow, and prints the results.  This command-line program makes it easy to debug and profile locally without starting the SQLFlow server and the Jupyter server.  We call this command-line program the SQLFlow REPL. SQLFlow REPL supports automatic code completion to ease applying the underlying powerful AI toolset of SQLFlow.
+We have a command-line tool called `sqlflow` which can access SQLFlow server like we do in Jupyter Notebook.  `sqlflow` works in the well known `REPL` mode, that is to say, reads SQL statements from the console, evaluates them by calling SQLFlow server, and prints the results.  `sqlflow` supports automatic code completion to ease applying the underlying powerful AI toolset of SQLFlow.
 
 ![](figures/repl.gif)
 
 ## Quick Start
 
-The SQLFlow Docker image contains the REPL command-line program.  We can run an example session by typing the following command on MacOS.  If you run Docker on Linux, please change `host.docker.internal:3306` to `localhost:3306`.
+The SQLFlow Docker image contains the `sqlflow` client.  We can run an example session by typing the following command on MacOS.  If you run Docker on Linux, please change `host.docker.internal:3306` to `localhost:3306`.
 
 ```
 docker run -it --rm --net=host sqlflow/sqlflow bash -c '/start.sh populate-example-dataset-mysql-local; \
```

Comment:
`--net=host` doesn't work on Docker Desktop, how about changing it to port mapping?
Correct me if my memory fails me.

Reply:
@leomzhong Do we need a ; between /start.sh and populate-example-dataset-mysql-local?

Reply:
> @leomzhong Do we need a ; between /start.sh and populate-example-dataset-mysql-local?

`populate-example-dataset-mysql-local` is a command-line option of `/start.sh`

Reply:
> @leomzhong Do we need a ; between /start.sh and populate-example-dataset-mysql-local?

No, `populate-example-dataset-mysql-local` is a param for the `start.sh` sell script.